apps/site/next-data/providers/blogData.ts

```diff
@@ -2,15 +2,22 @@ import { cache } from 'react';
 
 import generateBlogData from '@/next-data/generators/blogData.mjs';
 import { BLOG_POSTS_PER_PAGE } from '@/next.constants.mjs';
-import type { BlogCategory, BlogPostsRSC } from '@/types';
+import type { BlogCategory, BlogPostsRSC, BlogPost } from '@/types';
 
-const { categories, posts } = await generateBlogData();
+let blogData: {
```

Comment:
Create a type for this, no inline types.

Reply:
change reverted, no longer needed (thanks to work done on the adapter 🙂) 🥳